.vsts.release.yml

```diff
@@ -295,3 +318,29 @@ stages:
         }
         Invoke-RestMethod @assetsParams
       displayName: Create agent release on Github
+
+- stage: CreatePRs
+  dependsOn: Release
+  jobs:
+  ################################################################################
+  - job: create_ado_prs
+  ################################################################################
+    displayName: Create PRs in AzureDevOps
+    pool:
+      vmImage: ubuntu-16.04
```

Comment:
doesn't matter a whole lot, but maybe we should run these on 18.04 instead of 16.04?

Reply:
Done